tests/testthat/test-subscript-loc.R

```diff
@@ -410,6 +427,10 @@ test_that("conversion to locations has informative error messages", {
     "# num_as_location() optionally forbids negative indices"
     num_as_location(dbl(1, -1), 2L, negative = "error")
 
+    "# num_as_location() optionally forbids zero indices"
+    num_as_location(0L, 1L, zero = "error")
```

Comment:
The header should correspond to a `test_that()` block containing `expect_error()` statements for the tested expressions, wrapped within `verify_errors()`.

This way there is a hard check on the expected error in addition to the output check, and the `verify_errors()` reminds us to update the corresponding block of expressions within `verify_output()`.